### PR TITLE
Modified the acquisition value title to reflect the changes in prototype

### PR DESCRIPTION
**Pull request by euptonqa**

### CODE REVIEW

File: app/views/calculation/resident/properties/gain/acquisitionValue.scala.html [hunk 1]
```diff
@@ -6,15 +6,23 @@
...
-</a>
+    <a id="lossesLink" class="external-link" rel="external" href="https://www.gov.uk/capital-gains-tax/losses"
+       target="_blank" data-journey-click="help:govUK:rtt-properties-acquisitionCgtLosses">
+        @Messages("calc.resident.acquisitionValue.bullet.three.link")
+        <span>@Messages("calc.base.externalLink")</span>
+    </a>
+}
+
+@marketValueLink = {
+    <a id="mrktValLink" class="external-link" rel="external" href="https://www.gov.uk/capital-gains-tax/market-value"
```
Comment (miranda-hawkes) [inline app/views/calculation/resident/properties/gain/acquisitionValue.scala.html]: Just being picky, can this not be abbreviated and be called 'marketValueLink'?

File: test/views/resident/properties/gain/AcquisitionValueViewSpec.scala [hunk 2]
```diff
@@ -96,8 +96,11 @@ class AcquisitionValueViewSpec extends UnitSpec with WithFakeApplication with Fa
 
       "has additional content that" should {
 
-        s"have a bullet point list title of ${messages.bulletListTitle}" in {
+       /* s"have a bullet point list title of ${messages.bulletListTitle}" in {
```
Comment (miranda-hawkes) [inline test/views/resident/properties/gain/AcquisitionValueViewSpec.scala]: Could you remove this commented out part please?

File: test/views/resident/properties/gain/AcquisitionValueViewSpec.scala [hunk 3]
```diff
@@ -96,8 +96,11 @@ class AcquisitionValueViewSpec extends UnitSpec with WithFakeApplication with Fa
...
           doc.select("div.indent p#bullet-list-title").text() shouldEqual messages.bulletListTitle
+        }*/
+        s"have a bullet point list title of ${messages.bulletListTitleOne + messages.bulletListTitleTwo + messages.bulletListTitleThree}" in {
+          doc.select("div.indent p#bullet-list-title").text() shouldEqual (messages.bulletListTitleOne + messages.bulletListTitleTwo+messages.bulletListTitleThree)
```
Comment (miranda-hawkes) [inline test/views/resident/properties/gain/AcquisitionValueViewSpec.scala]: Could you also add a test for the external link for market value?
E.g.
doc.getElementById("mrktValLink").attr("href") shouldBe _url_

**Comment by euptonqa** [comment]
For some reason one of the tests for an unrelated/out-of-scope controller I haven't touched isn't passing; I'm assuming this is probably an issue with my local environment?

**Comment by euptonqa** [comment]
[info] *** 1 TEST FAILED ***
[error] Failed: Total 3889, Failed 1, Errors 0, Passed 3888
[error] Failed tests:
[error] 	controllers.resident.shares.ReportControllerSpec.GainSummaryActionSpec
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 37 s, completed 06-Sep-2016 09:35:24

File: test/views/resident/properties/gain/AcquisitionValueViewSpec.scala [hunk 4]
```diff
@@ -96,10 +96,12 @@ class AcquisitionValueViewSpec extends UnitSpec with WithFakeApplication with Fa
 
       "has additional content that" should {
 
-        s"have a bullet point list title of ${messages.bulletListTitle}" in {
-          doc.select("div.indent p#bullet-list-title").text() shouldEqual messages.bulletListTitle
+        s"have a bullet point list title of ${messages.bulletListTitleOne + messages.bulletListTitleTwo + messages.bulletListTitleThree}" in {
+          doc.select("div.indent p#bullet-list-title").text() shouldEqual (messages.bulletListTitleOne + messages.bulletListTitleTwo+messages.bulletListTitleThree)
+        }
+        s"have an external link that points to the appropriate URL for the market value" in {
+          doc.getElementById("marketValueLink").attr("href")
```
Comment (miranda-hawkes) [inline test/views/resident/properties/gain/AcquisitionValueViewSpec.scala]: Missing the assertion here (shouldEqual) - e.g. the one on line 123 that checks the lossesLink

**Pull request merged**